pom.client.xml

```diff
@@ -766,13 +766,19 @@
       </profile>
 
       <profile>
-         <id>extra-modules</id>
+         <id>non-shipping-modules</id>
```

Comment:
Is there any way to include these modules by default but exclude them if some property is set and we can set that in the reporting step?

Reply:
You can - but that would mean that the default case would be to build unnecessary modules. I would prefer to opt-in to the non-shipping modules only when they are required for reporting.

Reply:
I guess I can live with this approach so long as the question about building other pom files that doesn't cause an error.